styleguide/guides/issues/security.rst

```diff
@@ -0,0 +1,12 @@
+🔐 Security
+===========
+
+TBD
+---
+- criticality (security)
```

Comment:
1, CVE if available
2. If it affects certain packages, add the package versions in the ticket